packages/cli/src/commands/extensions/install.test.ts

```diff
@@ -27,3 +33,97 @@ describe('extensions install command', () => {
     ).toThrow('Arguments source and path are mutually exclusive');
   });
 });
+
+describe('handleInstall', () => {
+  const consoleLogSpy = vi.spyOn(console, 'log');
+  const consoleErrorSpy = vi.spyOn(console, 'error');
+  const processSpy = vi
+    .spyOn(process, 'exit')
+    .mockImplementation(() => undefined as never);
+
+  afterEach(() => {
+    mockInstallExtension.mockClear();
```

Comment:
vi.resetAllMocks();

(prefer that instead of resetting each and every mock).

You probably needed `mockClear` because you didn't set up your mocks in a beforeEach